docker/ubuntu-based/prod/vswitch/Dockerfile

```diff
@@ -13,6 +13,16 @@ RUN apt-get update \
         netcat-openbsd \
  && rm -rf /var/lib/apt/lists/*
 
+# install ETCD
+ENV ETCD_VER=v3.3.9
+ENV GOOGLE_URL=https://storage.googleapis.com/etcd
+ENV GITHUB_URL=https://github.com/coreos/etcd/releases/download
+ENV DOWNLOAD_URL=${GOOGLE_URL}
+
+RUN curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+RUN tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/etcd --strip-components=1
```

Comment:
Put all RUN commands into one so that intermediate layers are not produced.

Reply:
We do we need to install ETCD anyway? My understanding is that we are using etcdctl for the init container from quay.io/coreos/etcd:v3.1.10

Reply:
Leftovers from testing, shouldn't be there since I moved to the quay.io/coreos/etcd:v3.1.10 removing....